plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -402,7 +402,7 @@ public void updateViewColumnComment(ConnectorSession session, SchemaTableName sc
     private SessionCatalog.SessionContext convert(ConnectorSession session)
     {
         return switch (sessionType) {
-            case NONE -> SessionCatalog.SessionContext.createEmpty();
+            case NONE -> new SessionContext(randomUUID().toString(), null, null, ImmutableMap.of(), session.getIdentity());
```

Comment:
previously we weren't passing session identity here. is it OK to pass it now?

Reply:
We're only doing a pass-through for Trino for the file system, but not populating the identity for Iceberg. The other arguments are the same as you get from `SessionContext.createEmpty()`.